<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EocFreq</class>
 <widget class="QWidget" name="EocFreq">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>48</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="freqRestore">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore</string>
   </property>
  </widget>
  <widget class="QPushButton" name="freqSave">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="freqLoad">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="freqLog">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>70</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </widget>
  <widget class="QComboBox" name="freqFC">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>fc</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="freqFullBW">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>bw</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="freqBW">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>bw divi ratio</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="freqRange">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>140</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
